packages/react-server-webpack-plugin/README.md

```diff
@@ -0,0 +1,152 @@
+# `@shopify/react-server-webpack-plugin`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-server-webpack-plugin.svg)](https://badge.fury.io/js/%40shopify%2Freact-server-webpack-plugin.svg)
+
+A webpack plugin which generates entrypoints for `@shopify/react-server` based applications. This plugin allows you to run a universal React application without needing any client/server-specific code.
```

Comment:
Perhaps adding something like "generates in-memory entrypoints" or "at runtime" in here somewhere.